modules/rostests/rosautotest/main.cpp

```diff
@@ -8,6 +8,7 @@
 #include "precomp.h"
 #include <cstdio>
 DWORD TestStartTime;
+WCHAR TestName[];
```

Comment:
Missing length.

Reply:
This was done purposefully here to indicate that this is only a forward define and not the primary definition which is actually in precomp.h.